templates/zerver/api/dev-fetch-api-key.md

```diff
@@ -19,9 +19,7 @@
 
 #### Return values
 
-* `api_key`: The API key that can be used to authenticate as the requested
-    user.
-* `email`: The email address of the user who owns the API key.
+{generate_return_values_table|zulip.yaml|/dev_fetch_api_key:post}
 
 #### Example response
 
```

Comment:
This is a side comment, but we're not far from being able to have this entire file be deleted and generated entirely by OpenAPI data; only the page title and filename aren't coming from there in a predictable way :).